### PR TITLE
[REF] Support of 'noupdate' in check 'duplicate-xml-record-id'

### PR DESCRIPTION
**Pull request by moylop260**
Many times we need duplicate a xml_id one with noupdate="1" and another one
with noupdate="0"
Fix OCA/maintainer-quality-tools#276

### CODE REVIEW

**Comment by guewen** [comment]
:+1: Thanks!

File: pylint_odoo/misc.py [hunk 1]
```diff
@@ -205,5 +205,10 @@ def get_xml_record_ids(self, xml_file, module=None):
                 else [self.module, record.get('id')]
             if module and xml_module != module:
                 continue
-            xml_ids.append(xml_module + '.' + xml_id)
+            # Support case where use two xml_id:
```
Comment (pedrobaeza) [inline pylint_odoo/misc.py]: s/use/using

**Comment by pedrobaeza** [comment]
Little nitpicking, but :+1:

File: pylint_odoo/misc.py (hunk 1 above)
Comment (moylop260) [inline pylint_odoo/misc.py]: @pedrobaeza 
Thank you
Fixed!

**Comment by moylop260** [comment]
Fixed typo

**Comment by dreispt** [comment]
:+1: for this.

This check should also be done in CSVs, at least for Security ACL files following the standard columns structure: I recently saw a bug fix where the lines where duplicated but the author forgot to change the ID of the second line, so it overwrites the former permission and the module doesn't work.

**Comment by moylop260** [comment]
@dreispt 
Good point
Could you create a new issue to have a ToDo list?

**Pull request merged**